vocabularies/qld-resource-permit-status.ttl

```diff
@@ -0,0 +1,214 @@
+    skos:broader gmps:submitted ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Awaiting payment confirmation"@en .
+
+gmps:competing-application a skos:Concept ;
+    skos:definition "This application represents a competing application."@en ;
+    skos:broader gmps:application,
+        gmps:granted ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Competing application"@en .
+
+gmps:conditionally-surrendered a skos:Concept ;
+    skos:definition "The application has been conditionally surrendered."@en ;
+    skos:broader gmps:application,
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Conditionally surrendered"@en .
+
+gmps:conversion-of-ml-to-mc a skos:Concept ;
+    skos:definition "The Mining Lease has been converted to a Mining Claim."@en ;
+    skos:broader gmps:granted ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Conversion of ML to MC"@en .
+
+gmps:expired a skos:Concept ;
+    skos:definition "The Resource Permit has expired."@en ;
+    skos:broader gmps:granted,
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Expired"@en .
+
+gmps:finalised a skos:Concept ;
+    skos:definition "The Resource Permit has no outstanding requirements."@en ;
+    skos:broader gmps:non-current ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Finalised"@en .
+
+gmps:in-progress a skos:Concept ;
+    skos:definition "The Application is in-progress."@en ;
+    skos:broader gmps:discarded,
+        gmps:prelodgement ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "In progress"@en .
+
+gmps:on-hold-oil-shale-application a skos:Concept ;
+    skos:definition "The oil-shale application has been placed on-hold."@en ;
+    skos:broader gmps:application ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "On-hold oil shale application"@en .
+
+gmps:paid a skos:Concept ;
+    skos:definition "Application payment has been made and confirmation of payment has been received."@en ;
+    skos:broader gmps:discarded,
+        gmps:submitted ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Paid"@en .
+
+gmps:payment-authorised a skos:Concept ;
+    skos:definition "Payment authorisation has been requested."@en ;
+    skos:broader gmps:prelodgement,
+        gmps:submitted ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Payment authorised"@en .
+
+gmps:payment-pending a skos:Concept ;
+    skos:definition "Payment is pending."@en ;
+    skos:broader gmps:discarded,
+        gmps:submitted ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Payment pending"@en .
+
+gmps:preferred-tenderer a skos:Concept ;
+    skos:definition "The applicant is the preferred tenderer."@en ;
+    skos:broader gmps:application,
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Preferred Tenderer"@en .
+
+gmps:priority-applicant a skos:Concept ;
```

Comment:
This is a status for an applicant, not a permit or permit application

Reply:
Is this supposed to be priority application rather than applicant? If this is relative to a ranked application the definition of each should reference the other so the relationship and distinction is clear.